resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,426 @@
+						{{-- Generate Load Figures --}}
+						<div class="row">
+						@php $LoadArray = [] ; @endphp
+						@foreach($SUB->fares as $SUBfares)
+							@if($SUBfares->capacity > 0)
+								@php 
+									$RandomLoadPerFare = ceil(($SUBfares->capacity * $FKRandomLoad) /100);
+									$LoadArray[] = ['SeatType' => $SUBfares->code];
+									$LoadArray[] = ['SeatLoad' => $RandomLoadPerFare];
+								@endphp
+								<div class="col-sm-4">
+									<label for="LoadFare{{ $SUBfares->id }}">{{ $SUBfares->name }} Load [ Max: {{ number_format($SUBfares->capacity) }} ]</label>
+									<input id="LoadFare{{ $SUBfares->id }}" type="text" class="form-control" value="{{ number_format($RandomLoadPerFare) }}" disabled/>
+								</div>
+							@endif
+						@endforeach
+						@php 
+							$LoadCollection = collect($LoadArray) ; 
+							$TotalGenLoad = $LoadCollection->sum('SeatLoad') ;
+						@endphp
+						</div>
+						{{-- End Generate Load Figures --}}
+												
+						@php $PxWeight = '208' ; @endphp {{-- Just For Safety  --}}
+						@if($TotalGenLoad < '900')
+							{{-- >Passenger Flight --}}
+							@if($flight->flight_type == 'C')
+								<input type="hidden" name="acdata" value="{'paxwgt':197}"> {{-- Use and Send Charter Pax Weights --}}
+								@php $PxWeight = '197' ; @endphp
+							@else
+								<input type="hidden" name="acdata" value="{'paxwgt':219}"> {{-- Use and Send Scheduled Pax Weights Type J/G and all the rest --}}
+								@php $PxWeight = '219' ; @endphp											
+							@endif
+							<br>
+							<div class="row">
+								<div class="col-sm-4">
+									@if(setting('units.weight') === 'kg')
+										@php $EstimatedPayload = number_format(round(($PxWeight * $TotalGenLoad) / 2.2)) ; @endphp
+									@else
+										@php $EstimatedPayload = number_format(round($PxWeight * $TotalGenLoad)) ; @endphp
+									@endif
+									<label for="EstimatedLoad">Estimated Load For {{ $TotalGenLoad }} Pax</label>
+									<input id="EstimatedLoad" type="text" class="form-control" value="{{ $EstimatedPayload }} {{ setting('units.weight') }}" disabled/>
+								</div>
+							</div>
+							<input type="hidden" id="pax" name="pax" class="form-control" value="{{ $TotalGenLoad }}"/>
+						@else
+							{{-- This is A Cargo Flight So Send Pax 0 to avoid SimBrief auto generation --}}
+							<input type='hidden' id="pax" name='pax' value='0' maxlength='3'>
+							<input type='hidden' id="cargo" name='cargo' value="{{ $TotalGenLoad }}" maxlength='7'>		
+						@endif
+						@endif		
+					@endforeach
+					{{-- END Get All Subfleets from flight And Generate Random Load For Each Fare Type of Selected SubFleet --}}
+					<br>
+						<div class="row">
+							@php
+							$FlightType = 'SimBrief Standart';
+							if($flight->flight_type == 'J') { $FlightType = 'Schedule All Adult Pax' ;}
```

Comment:
Use the ENUMs for these (look for `FlightType` being used elsewhere, it's better to use those constants than hardcoding the 'J', 'G', etc

Reply:
Please check the new commit.